src/main/resources/swagger/api-docs.yaml

```diff
@@ -211,6 +211,43 @@ paths:
           description: Workspace does not exist
         500:
           description: Internal Server Error
+  /workspaces/{workspaceNamespace}/{workspaceName}/entities/copy:
+    post:
+      tags:
+      - Entities
+      operationId: copyEntities
+      summary: |
+        Copy entities from one workspace to another
+      produces:
+        - application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: body
```

Comment:
if the body of this request contains both the source and destination workspaces, why duplicate the namespace/name in the url as well? Ideally, we'd remove it from one of those locations. At minimum, specify in the swagger doc if the url params are source or destination workspace.